string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,21 @@
+/* exported isVowel */
+
+function isVowel(char) {
+  char = char.charAt(0);
+
+  switch (char) {
+    case 'a':
+    case 'e':
+    case 'i':
+    case 'o':
+    case 'u':
+    case 'A':
+    case 'E':
+    case 'I':
+    case 'O':
+    case 'U':
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
If we convert `char` to lowercase, we can simplify this a bit. That way we only need to worry about checking lowercase and not uppercase as well.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```

Reply:
understood!